exchange/huobi/signer.go

```
@@ -15,7 +16,9 @@ type Signer struct {
 
 func (self Signer) Sign(msg string) string {
 	mac := hmac.New(sha256.New, []byte(self.Secret))
-	mac.Write([]byte(msg))
+	if _, err := mac.Write([]byte(msg)); err != nil {
+		log.Printf("Encode message error: %s", err.Error())
```

this function should return an error?